parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -34,7 +34,7 @@ ExecStart=/usr/local/bin/kubelet \
         --v=2 {{if NeedsContainerd}}--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock{{end}} \
         --volume-plugin-dir=/etc/kubernetes/volumeplugins \
         {{- if IsDynamicKubeletEnabled}}
-        --config /etc/default/kubeletconfig.json --dynamic-config-dir /etc/default/dynamickubelet \
+        --config /etc/default/kubeletconfig.json \
         {{- end}}
         $KUBELET_FLAGS \
```

Comment:
do we need a fallback dynamic-config-dir, just in case that it's not passed in by RP?

Reply:
I think we should backfill the toggle in RP so it always pass in the flag.

Reply:
The RP always pass in the flag. My concern is not on RP but just for absolute safety in agentbaker side - imagine some other tool uses agentbaker and forgets to pass in  the dir, what can we do?